src/classes/EP_Task_UTIL.cls

```diff
@@ -113,35 +116,39 @@ public class EP_Task_UTIL {
         initializeMaps(templateIds);
     }
 
-
     /*******************************************************************************************************
     * @description Queries and builds maps for templates and template tasks from a set of template Ids.
     * @param templateIds a set of Engagement Plan Template Ids to query and build maps for.
     * @return void This method instantiates class properties.
     ********************************************************************************************************/
     public void initializeMaps(Set<Id> templateIds) {
+        Set<String> addlEPTaskFields = this.getAdditionalFieldMappings();
+        Set<String> epTaskFields = getPrimaryEPTaskFields();
+        // Determine what additional fields need to be added to the task query, if any
+        addAdditionalEPTaskFieldsIfNotPresent(epTaskFields, addlEPTaskFields);
```

Comment:
Since the `addlEPTaskFields`, ie `additionalFieldMappings` is not used in this method, then it can be called in `addAdditionalEPTaskFieldsIfNotPresent` without being passed as the input argument. So lines 125-128 can be replaced with
```
Set<String> epTaskFields = getPrimaryEPTaskFields();
epTaskFields.addAll(getAdditionalEPTaskFieldsIfNotPrimary().values());
```

and lines 174-183 can be replacced with 
```
taskEPTaskFieldMap.putAll(getAdditionalEPTaskFieldsIfNotPrimary());
```

where `addAdditionalEPTaskFieldsIfNotPresent` is replaced with the following method:
```
private Map<String, String> getAdditionalEPTaskFieldsIfNotPrimary() {
        Map<String, String> fields = new Map<String, String>();
        Set<String> primaryFields = toLowerCase(getPrimaryEPTaskFields());

        for (String fieldName : additionalEPTaskFields) {
            if (!primaryFields.contains(fieldName.toLowerCase())) {
                fields.put(fieldName, fieldName);
            }
        }

        return fields;
    }
```